src/lib/Libpbs/pbs.pc.in

```diff
@@ -4,7 +4,7 @@ libdir=@libdir@
 includedir=@includedir@
 
 Name: @PACKAGE_NAME@
-Description: Library for the PBSPro Workload Manager
+Description: Library for the OpenPBS Workload Manager
```

Comment:
Since this is a description and not a package name, suggest "PBS" here rather than "OpenPBS"